hack/cluster-deploy.sh

```diff
@@ -104,9 +104,11 @@ fi
 
 _deploy_infra_for_tests
 
-# TODO: Remove it when cdi is supported on ARM
-if [[ ${ARCHITECTURE} != *aarch64 ]]; then
-    _ensure_cdi_deployment
+if [[ -n $DEPLOY_CDI ]]; then
```

Comment:
I think that we should add such an option to allow an op-out instead of opt-in to keep things working as they are for everyone right now.

Reply:
Done